stubs/pyOpenSSL/OpenSSL/SSL.pyi

```diff
@@ -115,25 +117,63 @@ def SSLeay_version(type: int) -> str: ...
 class Session: ...
 
 class Connection:
-    def __getattr__(self, name: str) -> Any: ...  # incomplete
+    def __getattr__(self, name: str) -> Any: ...  # takes attributes from `self._socket`
     def __init__(self, context: Context, socket: socket.socket | None = ...) -> None: ...
+    def get_context(self) -> Context: ...
+    def set_context(self, context: Context) -> None: ...
+    def get_servername(self) -> bytes | None: ...
+    def set_tlsext_host_name(self, name: bytes) -> None: ...
+    def pending(self) -> int: ...
+    def send(self, buf: ReadableBuffer, flags: int = ...) -> int: ...
+    write = send
+    def sendall(self, buf: ReadableBuffer, flags: int = ...) -> int: ...
+    def recv(self, bufsiz: int, flags: int | None = ...) -> bytes: ...
+    read = recv
+    def recv_into(self, buffer: bytearray, nbytes: int | None = ..., flags: int | None = ...) -> int: ...
     def connect(self, addr: str | bytes | Sequence[str | int]) -> None: ...
+    def connect_ex(self, addr: _Address | bytes) -> int: ...
+    def accept(self) -> tuple[Connection, _RetAddress]: ...
+    def shutdown(self) -> bool: ...
     def do_handshake(self) -> None: ...
-    def get_peer_certificate(self) -> X509: ...
-    def set_tlsext_host_name(self, name: bytes) -> None: ...
+    def get_certificate(self) -> X509 | None: ...
+    def get_peer_certificate(self) -> X509 | None: ...
+    def get_peer_cert_chain(self) -> list[X509] | None: ...
+    def get_verified_chain(self) -> list[X509] | None: ...
     def bio_read(self, bufsiz: int) -> bytes: ...
     def bio_write(self, buf: bytes) -> int: ...
-    def recv(self, bufsiz: int, flags: int | None = ...) -> bytes: ...
-    def sendall(self, buf: bytes, flags: int = ...) -> int: ...
+    def bio_shutdown(self) -> None: ...
+    def renegotiate(self) -> bool: ...
+    def renegotiate_pending(self) -> bool: ...
+    def total_renegotiations(self) -> int: ...
+    def makefile(self, *args: Any, **kwargs: Any) -> NoReturn: ...
     def set_accept_state(self) -> None: ...
     def set_connect_state(self) -> None: ...
-    def get_peer_cert_chain(self) -> list[X509]: ...
-    def get_alpn_proto_negotiated(self) -> bytes: ...
+    def get_client_ca_list(self) -> list[X509Name]: ...
+    def get_cipher_list(self) -> list[str]: ...
     def get_cipher_name(self) -> str | None: ...
+    def get_cipher_bits(self) -> int | None: ...
+    def get_cipher_version(self) -> str | None: ...
     def get_protocol_version_name(self) -> str: ...
+    def get_protocol_version(self) -> int: ...
     def get_shutdown(self) -> int: ...
+    def set_shutdown(self, state: int) -> None: ...
+    def get_state_string(self) -> bytes: ...
+    def server_random(self) -> bytes: ...
+    def client_random(self) -> bytes: ...
+    def master_key(self) -> bytes: ...
+    def export_keying_material(self, label: Any, olen: Any, context: Any = ...) -> Any: ...  # TODO: type, see RFC-5705
```

Comment:
Why annotate these if we're adding the TODO comment? Why not just leave it unannotated, which serves as its own implicit TODO?

Reply:
I don't know 🙂 
pyright was failing without them:
- https://github.com/python/typeshed/runs/7480728281?check_suite_focus=true
- https://github.com/python/typeshed/runs/7480713768?check_suite_focus=true

```
  /home/runner/work/typeshed/typeshed/stubs/pyOpenSSL/OpenSSL/SSL.pyi:166:38 - error: Type of parameter "label" is unknown (reportUnknownParameterType)
  /home/runner/work/typeshed/typeshed/stubs/pyOpenSSL/OpenSSL/SSL.pyi:166:45 - error: Type of parameter "olen" is unknown (reportUnknownParameterType)
  /home/runner/work/typeshed/typeshed/stubs/pyOpenSSL/OpenSSL/SSL.pyi:166:9 - error: Return type is unknown (reportUnknownParameterType)
```

Reply:
You can exclude the file in `pyrightconfig.stricter.json`.

Reply:
Or we can use `_typeshed.Incomplete`, which is an alias for `Any` but serves as an easily greppable TODO marker :)